frontend/src/app/forms/steps/pictures-entry/pictures-entry.component.html

```diff
@@ -0,0 +1,45 @@
+<div class="main-description">
+  <h1>Picture</h1>
+
+  <!--<ion-img [src]="photo"></ion-img>-->
+  <div
+    *ngIf="
+      formsData.controls.pictureEntry.controls.picString.invalid;
+      else elseBlock
+    "
+  >
+    <span>
+      Add a picture of yourself by tapping on the image icon.
+    </span>
+    <ion-button
+      (click)="takePicture()"
+      class="custom-button"
+      fill="clear"
+      size="large"
+    >
+      <ion-icon name="person-circle"></ion-icon>
+    </ion-button>
+  </div>
+  <ng-template #elseBlock>
+    <span>
+      Good job! If you want to take another picture, just delete the actual
+      picture by sliding on it and take another one.
+    </span>
+    <ion-item-sliding class="ion-no-padding custom-sliding">
```

Comment:
Instead of sliding the item to delete it, maybe just show a button? Sliding such a large item to delete is pretty uncommon I guess... I would just show a "delete" button somewhere. Maybe just a trash icon.

What do you think?

Reply:
I know that this is not part of my mockups. Would it help if I added something like this?

Reply:
yeah a little mockup where trash button should then be can help.
And yeah maybe sliding is for the picture not rly goo here.

Reply:
I have now created the mockups for it. Just click on the round button in the middle in order to see the changes.